quic-client/src/quic_client.rs

```diff
@@ -68,7 +68,7 @@ impl AsyncTaskSemaphore {
 lazy_static! {
     static ref ASYNC_TASK_SEMAPHORE: AsyncTaskSemaphore =
         AsyncTaskSemaphore::new(MAX_OUTSTANDING_TASK);
-    static ref RUNTIME: Runtime = tokio::runtime::Builder::new_multi_thread()
+    pub static ref RUNTIME: Runtime = tokio::runtime::Builder::new_multi_thread()
```

Comment:
The idea is to use the same runtime in `ConnectionCache` as in the `tpu-client-next`. Since blocking `ConnectionCache` uses this `RUNTIME` static, the less invasive was to make it `pub` and use where needed.